apps/nextra/pages/en/build/cli/trying-things-on-chain/ledger.mdx

```diff
@@ -117,3 +119,495 @@ yes
+
+Since the CLI profile `ledger-wallet-1000-recovered` is now stale, rename it
+in case you get interrupted and forget that the private key has been rotated:
+
+```sh filename="Terminal"
+aptos config rename-profile \
+    --profile ledger-wallet-1000-recovered \
+    --new-profile-name ledger-wallet-1000-stale
+```
+
+<details>
+<summary>Example output</summary>
+```sh filename="Terminal"
+{
+  "Result": "Renamed profile ledger-wallet-1000-recovered to ledger-wallet-1000-stale"
+}
+```
+</details>
+
+### Rotate back to Ledger
+
+Once you've signed the large package publication transaction with the hot key,
+you can then rotate the authentication key back to to the corresponding to the
+private key on the Ledger at index 1000:
+
+```sh filename="Terminal"
+aptos account rotate-key \
+    --assume-yes \
+    --new-derivation-index 1000 \
+    --profile temporary-hot-wallet \
+    --save-to-profile ledger-wallet-1000
+```
+
+Follow the instructions from the CLI prompt:
+
+```sh filename="Terminal"
+Approve rotation proof challenge signature on your Ledger device
+```
+
+<details>
+<summary>Example output</summary>
+```sh filename="Terminal"
+{
+  "Result": {
+    "message": "Saved new profile ledger-wallet-1000",
+    "transaction": {
+      "transaction_hash": "0x9503819d4ea13bcd9eafed25984807d86d22e8a9837565a7495b54d13890d103",
+      "gas_used": 449,
+      "gas_unit_price": 100,
+      "sender": "aaac71af5f2a4af4ec2639a15799bf9b945afb061c8bee102b636531c1b00eb5",
+      "sequence_number": 2,
+      "success": true,
+      "timestamp_us": 1717986672963544,
+      "version": 742,
+      "vm_status": "Executed successfully"
+    }
+  }
+}
+```
+</details>
+
+Verify that the `ledger-wallet-1000-stale` and `ledger-wallet-1000` profiles
+have the same `account` address and `public_key`:
+
+```sh filename="Terminal"
+aptos config show-profiles --profile ledger-wallet-1000-stale
+aptos config show-profiles --profile ledger-wallet-1000
+```
+
+Delete the `temporary-hot-wallet` and `ledger-wallet-1000-stale` profiles,
+which you no longer need.
+
+```sh filename="Terminal"
+aptos config delete-profile --profile temporary-hot-wallet
+aptos config delete-profile --profile ledger-wallet-1000-stale
+```
+
+<details>
+<summary>Example output</summary>
+```sh filename="Terminal"
+{
+  "Result": "Deleted profile temporary-hot-wallet"
+}
+{
+  "Result": "Deleted profile ledger-wallet-1000-stale"
+}
+```
+</details>
+
+Since you no longer need the temporary private key, delete it too.
+
+<Callout type="info" emoji="🧠">
+If you are using a UNIX-like machine:
+
+```shell filename="Terminal"
+rm private-key-*
+```
+</Callout>
+
+### Clean up
+
+Delete the remaining test profile:
+
+```shell filename="Terminal"
+aptos config delete-profile --profile ledger-wallet-1000
+```
+
+Then stop the localnet.
+
+<Callout type="info" emoji="🧠">
+If you are using a UNIX-like machine:
+
+```shell filename="Terminal"
+aptos config delete-profile --profile ledger-wallet-1000
```

Comment:
This is also not a command?

Reply:
As I mentioned in the PR description

> Install the CLI from source, as described in https://github.com/aptos-labs/aptos-core/pull/11151

This PR pairs simply because I had to split docs and source into two separate repos https://github.com/aptos-labs/aptos-core/pull/11151

Reply:
Noting that I've updated the PR description per the recent set of `aptos-core` PRs that have merged, to stipulate installing the latest Framework updates